client/css/style.css

```diff
@@ -2312,11 +2312,7 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 
 	#viewport.menu-open #sidebar-overlay,
 	#viewport.menu-dragging #sidebar-overlay {
-		display: block;
-	}
-
-	#viewport.menu-open .messages {
-		pointer-events: none;
```

Comment:
This is not necessary anymore since overlay eats all clicks. Thankfully even on iOS, only scroll events go through, but clicks are eaten just fine.